layouts/shortcodes/observability_pipelines/configure_log_archive/google_cloud_storage/instructions.md

```diff
@@ -0,0 +1,36 @@
+### Create a storage bucket
+
+1. Navigate to [Google Cloud Storage][9091].
+1. On the Buckets page, click **Create** to create a bucket for your archives..
+1. Enter a name for the bucket and choose where to store your data.
+1. Select **Fine-grained** in the **Choose how to control access to objects** section.
+1. Do not add a retention policy because the last data needs to be rewritten in some rare cases (typically a timeout case).
```

Comment:
```suggestion
1. Do not add a retention policy because the most recent data needs to be rewritten in some rare cases (typically a timeout case).
```